src/components/VmDetails/cards/NicsCard/style.css

```diff
@@ -24,11 +24,24 @@
 
 .ip4-container,
 .ip6-container {
+  padding: 0 5px 3px 5px;
   font-size: 10px;
-  padding: 0 10px;
+  flex: 1 0 33.3333%;
+  max-width: 100%;
+  min-width: 80px;
 }
+
 .ip6-container {
-  padding-left: 0;
+  flex: 2 0 66.6667%;
+}
+
+.ip + .ip {
+  padding-bottom: 3px;
```

Comment:
Since the rule can be interpreted as "any .ip that comes after an .ip" or "all .ip in a row except the first one", you'll want to use `padding-top` instead of `padding-bottom`